cloudynitzar.sh

```diff
@@ -34,8 +34,17 @@ CLOUDYPACKAGES="cDistro.chroot avahi-ps.chroot serf.chroot ipfs.chroot"
 OLDCLOUDYPACKAGES="getinconf-client.chroot"
 
 # List of required Debian/Ubuntu packages for the Cloudynization process
-DEBREQPACKAGES="curl unzip dirmngr"
 
+case $RELEASENAME in
```

Comment:
The same thing is actually done with the per-distribution [lbmake hooks](https://github.com/Clommunity/lbmake/tree/master/hooks); take a look some lines below.

I agree your solution is more elegant than the messy lbmake thing but let's assume it's there for a reason and just add the missing distributions (e.g., [Bionic Beaver](https://github.com/Clommunity/lbmake/commit/61f663794597ee8c8062812fff675aef895fde18))

Reply:
Ok,  I will do it as you told me. Thanks!

Reply:
is it working? https://github.com/Clommunity/lbmake/blob/master/packages-bionic#L6

Reply:
Yes, it is :+1: